src/modules/launcher/Wox.Plugin/IPublicAPI.cs

```diff
@@ -43,7 +43,7 @@ public interface IPublicAPI
         /// <summary>
         /// Restart Wox
         /// </summary>
-        void RestarApp();
+        void RestartApp();
```

Comment:
This one worried me when I saw it a while back since it was on the interface for the wox main interface. No matter what, we would have to create a shim to work with their plugins.

Reply:
Thanks. I'll split off anything that requires significant change/discussion. #3774